apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java

```diff
@@ -21,11 +27,26 @@
 public final class ApolloConverterFactory extends Converter.Factory {
   private final Map<Type, ResponseFieldMapper> responseFieldMappers;
   private final Map<ScalarType, CustomTypeAdapter> customTypeAdapters;
+  private final Moshi moshi;
```

Comment:
I'm not crazy about having a dependency on moshi. I still feel like gson is the most popular converter on android. Having both including in classpath is extraneous methods.  Maybe we can make a convert factory type setup similar to what retrofit does?